mods/d2k/sequences/infantry.yaml

```diff
@@ -196,10 +196,15 @@ medic: # actually thumper
 		Facings: -8
 		Transpose: true
 		Tick: 120
-	heal: DATA.R8
+	thump: DATA.R8
 		Start: 1458
 		Length: 5
 		Tick: 480
```

Comment:
This screenshot suggests that probably the idle animation isn't removed when deploying.